packages/larva/scripts/larva/larva.config.js

```diff
@@ -0,0 +1,42 @@
+const path = require( 'path' );
+
+module.exports = {
+	webpack: {
+		aliases: {
+			'@js': path.resolve( './src/js' )
+		},
+		entries: {
+			common: './entries/common.entry.js'
+		}
+	},
+
+	backstop: {
+		// testBaseUrl: 'https://notlaura.com',
```

Comment:
Do we still need this comment in here?

Reply:
Yeah - this is for toggling using a remote URL vs. the localhost – I need to document/organize this. Will do in later PR. Thanks!